### PR TITLE
2nd settings

### PR DESCRIPTION
**Pull request by garrettbolter**

### CODE REVIEW

File: views/components/form-holder.pug [hunk 1]
```diff
@@ -4,6 +4,6 @@ div(class='content-container intervalForm')
       include form-holder/interval-form.pug
       include form-holder/genre-form.pug
       include form-holder/decade-form.pug
-      include form-holder/theme-form.pug
+      //- include form-holder/theme-form.pug
```
Comment (nidhimurlidhar) [inline views/components/form-holder.pug:7]: Can you get rid of the commented out code?

File: client/main.js [hunk 2]
```diff
@@ -100,6 +106,58 @@ $(document).ready(function () {
         page.updateVolIcon(audioElement);
     });
 
+    // How I imagine this would work vvv, the console logs didn't even work
```
Comment (jasontyu) [inline client/main.js:109]: Just delete anything that doesn't work, so we don't get confused later.

File: client/main.js [hunk 3]
```diff
@@ -17,6 +17,11 @@ $(document).ready(function () {
     let genres = []
     let genrePreferences = [];
     let decadePreferences = [];
+    let settingsDisplay = $('#advSetForm');
+    let stopDisplay = $('#stop-timer');
+    let startDisplay = $('#start-timer');
+    let volumeDisplay = $('#vol-control');
+    let titleDisplay = $('#current-song-display');
```
Comment (jasontyu) [inline client/main.js:24]: What's the point of these, if they're not even used?

File: client/main.js [hunk 4]
```diff
@@ -89,6 +94,7 @@ $(document).ready(function () {
     }
 
     $('#mute').click(function () {
+        console.log("in mute");
```
Comment (jasontyu) [inline client/main.js:97]: Comment out or remove

File: client/main.js [hunk 5]
```diff
@@ -100,6 +106,58 @@ $(document).ready(function () {
...
+            titleDisplay[0].style["display"] = "none";
+        } else {
+            volumeDisplay[0].style["display"] = "none";
+            titleDisplay[0].style["display"] = "inline-block";
+        }  
+    });
+
     //Updates volume when slider is changed.
     $('#vol-control').on("input change", function () {
         tempVol = this.value;
```
Comment (jasontyu) [inline client/main.js:163]: (not related to `main.js`, I just can't leave a comment on a .png) Looks like `/public/images/officebeatz-banner copy.png` is a copy mistake, delete it?

File: views/components/about-tab.pug [hunk 6]
```diff
@@ -0,0 +1,15 @@
+div(id='about-tab', class='col s12')
+  div(class='content-container')
+    h5(style="margin: 20px; text-align: center") Below are a handful of scientific articles providing information on the negative cardiovascular effects of sitting.
+    h6(style="margin: 20px; text-align: center; color: grey") Keep in mind, this is not an exhaustive list. 
+    ul(style="margin: 20px")
+      li(style="margin: 10px 0")
```
Comment (jasontyu) [inline views/components/about-tab.pug:6]: Everything here should be using `class` to format, not `style`(that way you only need to change the style in one place - the css class)

File: views/components/form-holder.pug [hunk 7]
```diff
@@ -1,9 +1,9 @@
-div(class='content-container intervalForm')
+div(class='content-container intervalForm', id="advSetForm", style="display: none")
```
Comment (jasontyu) [inline views/components/form-holder.pug:1]: It's not good to have two IDs that are the same words (advanced settings form) but spelled differently. Can you find a way to not use one of the IDs, or perhaps choose a different word (eg advanced settings container)?

Comment (jasontyu) [reply views/components/form-holder.pug:1]: It doesn't look like id="advSetForm" is being used by anything

File: views/components/form-holder/decade-form.pug [hunk 8]
```diff
@@ -1,20 +1,17 @@
...
-        div(class="input-field inline col s12 m9 l4")
-          i(class="material-icons prefix") today
-          input(placeholder="Starting Year" id="startingDecade" type="number" min="1940" max="2019")
-      div(class="row")
-        div(class="input-field inline col s12 m9 l4")
-          i(class="material-icons prefix") today
-          input(placeholder="Ending Year" id="endingDecade" type="number" min="1940" max="2019")
-    
-    div(class="as-notes z-depth-1" id="decadeNotesContainer") 
+  div(class="", id="date-box  ")
```
Comment (jasontyu) [inline views/components/form-holder/decade-form.pug:6]: I think that if it only works when the ID is misspelled, then you should just remove the id tag completely (ie if spelling the ID correctly messes up the styling, then it shouldn't have the ID in the first place)

File: views/components/form-holder/decade-form.pug [hunk 9]
```diff
@@ -1,20 +1,17 @@
...
+    div(class="row")
+      div(class="input-field inline col s12 m9 l4")
+        i(class="material-icons prefix") Start
+        input(placeholder="Starting Year" id="startingDecade" type="number" min="1940" max="2019")
+      div(class="input-field inline col s12 m9 l4")
+        i(class="material-icons prefix") End
+        input(placeholder="Ending Year" id="endingDecade" type="number" min="1940" max="2019")
```
Comment (jasontyu) [inline views/components/form-holder/decade-form.pug:13]: [nit] the max should probably be 2020 (or find a way to dynamically fill in the current year)

File: views/components/form-holder/genre-form.pug [hunk 10]
```diff
@@ -1,86 +1,86 @@
 li(class="not-active")
-  div(class="collapsible-header valign-wrapper")
+  div(class="valign-wrapper")
     i(class="small material-icons valign-wrapper") music_note
-    | Genres
+    | &nbspGenres
```
Comment (jasontyu) [inline views/components/form-holder/genre-form.pug:4]: [nit] Do you need a space between &nbsp and Genres?

File: views/components/form-holder/interval-form.pug [hunk 11]
```diff
@@ -20,4 +20,7 @@ li(class="active")
...
\ No newline at end of file
+          input(placeholder="# of Minutes", id="customIntervalValue", type="number", min="1", max="120")
+  div(class="as-notes interval-notes z-depth-1", id="intervalNotesContainer") 
+    p(id="notes-header") Notes
+    p The more time you spend sitting each day- and the longer those periods of sitting are - the worse off your cardiovascular system is. Breaking up sitting with short, light-intensity physical activity is a healthy way to interrupt your sitting and make your blood vessels a bit more “happy." The green color indicates the ideal frequency for interrupting your sitting, the yellow is a moderate interval (less preferable, but not bad) to interrupt your sitting, and the red is a longer interval between sitting interruptions that is less preferred than the green or yellow. With that being said, any interruption is better than none, so don’t be worried if you can’t take a break from sitting every 15 or 30 minutes; just do your best to use the music as a positive reminder to get some movement in and help out your amazing body as often as you can : )
```
Comment (jasontyu) [inline views/components/form-holder/interval-form.pug:26]: [nit] space between day and -

File: views/components/home-tab.pug [hunk 12]
```diff
@@ -1,12 +1,13 @@
...
   div(class='content-container')
-    p(id='stuff') Welcome to <span id='accented'>#{title}</span>, a simple web application designed to get you out of your seat and on your feet. Prolonged sitting even for as little as one hour, damages the health of our blood vessels, and over time increases our risk of cardiovascular disease. Standing, stretching, or taking a short walk every 30-60 minutes can help to lessen the damaging effects of sitting.
+    p(id='stuff') Welcome to <span id='accented'>#{title}</span>, a simple web application designed to get you out of your seat and on your feet. Prolonged sitting even for as little as one hour, damages the health of our blood vessels, and over time increases our risk of cardiovascular disease. Standing, stretching, or taking a short walk every 15-45 minutes can help to lessen the damaging effects of sitting.
     br
     p(id='stuff') <span id='accented'>#{title}</span> will help improve your health by reminding you to get up and move through the power of your favorite songs. Unlike boring or annoying alarm reminders, OfficeBeatz, we hope, will facilitate a fun, energizing environment in your workplace, all while improving your health!
```
Comment (jasontyu) [inline views/components/home-tab.pug:12]: The ID shouldn't be `stuff` and there shouldn't be two elements with the same ID. Same with `accented`

File: views/components/new-header.pug [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+        li(class='tab')
+          a(href='#about-tab') About
+    div(id="covid")
+      p(id="covid-text") To help curb the spread of COVID 19 and keep you and your community safe, every time a song comes on, we encourage you to wash your hands for 20-seconds. Get some movement, and wash your hands. Stay healthy, stay safe &#9829
+    div(id="timer")
+      p(id="timer-time") 15:00:00
+      div
+        button(id="start-timer", class="timer-button") start
+        button(id="stop-timer", class="timer-button", style="display: none") stop
+        button(id="reset-timer", class="timer-button") reset
```
Comment (jasontyu) [inline views/components/new-header.pug:20]: The COVID text and timer stuff can be split out into its own components/pug template

File: views/components/song-controls.pug [hunk 14]
```diff
@@ -0,0 +1,7 @@
+//- need to mess with this class name to display & no slider
+div(class="100-wid")
+    div(class='title-container active', id="current-song-display")
+        a(class='songDisplay') Current song: 
+        a(id='titleID', class='songDisplay')
+        a(class='songDisplay')  by 
+        a(id='artistID', class='songDisplay')
```
Comment (jasontyu) [inline views/components/song-controls.pug:7]: Why are they `anchor` tags instead of `span`s?

File: views/index.pug [hunk 15]
```diff
@@ -1,12 +1,15 @@
...
+  //- include components/header.pug
+  include components/new-header.pug
 
   main
     include components/home-tab.pug
     include components/getting-started-tab.pug
-    include components/advanced-settings-tab.pug
+    include components/about-tab.pug
+    //- include components/advanced-settings-tab.pug
+    //- include components/new-settings-tab.pug
```
Comment (jasontyu) [inline views/index.pug:12]: If we're not using `new-settings-tab`, don't add it to the codebase?

File: client/main.js (hunk 3 above)
Comment (garrettbolter) [reply client/main.js:24]: I definitely use these

File: client/main.js (hunk 5 above)
Comment (garrettbolter) [reply client/main.js:163]: removed ty!

File: views/components/home-tab.pug (hunk 12 above)
Comment (garrettbolter) [reply views/components/home-tab.pug:12]: that's from the last group, i'll make a ticket for now

File: views/components/song-controls.pug (hunk 14 above)
Comment (garrettbolter) [reply views/components/song-controls.pug:7]: from last group, making ticket for now

**Comment by jasontyu** [comment]
(I noticed you missed a couple nits -- I think the comments were hidden for some reason -- but they're really minor so you can just grab em later).

**Pull request merged**